### DISCUSSION TITLE
2024-05-30

### DISCUSSION THREAD

**Discussion by jeasonstudio** [ant-design/ant-design-web3#917] [中文社区双周会]
上次双周会记录：https://github.com/ant-design/ant-design-web3/discussions/874

**Comment by jeasonstudio** [comment]
讨论 wagmi 配置的最佳实践：https://github.com/ant-design/ant-design-web3/pull/900#discussion_r1613405108
1. 用户在外部 `createConfig` 创建好传入（目前的方式）
2. 把 createConfig params 拆出来放到 WagmiProvider 上
3. 同时支持两种

**Comment by jeasonstudio** [comment]
pr 跟进：
* https://github.com/ant-design/ant-design-web3/pull/900
* https://github.com/ant-design/ant-design-web3/pull/901
* https://github.com/ant-design/ant-design-web3/pull/916

**Comment by jeasonstudio** [comment]
简单讨论，针对最新的 SocialFi 协议 Farcaster，我们有哪些可做的：https://github.com/ant-design/ant-design-web3/discussions/903

**Comment by yutingzhao1991** [comment]
- 官网进展
- Try it out 感觉可以提供更丰富的效果展示，吸引开发者使用
<img width="851" alt="image" src="https://github.com/ant-design/ant-design-web3/assets/1061968/51b11fce-762c-40e8-9661-8fcc640870aa">
主题看看能不能把 Ant Design 官网那一套搬过来 https://ant.design/index-cn

<img width="1549" alt="image" src="https://github.com/ant-design/ant-design-web3/assets/1061968/4bc63913-81e1-48d6-982c-09f874cfea52">

**Reply by jeasonstudio** [reply]
联系设计师把 try it out 区块做的好看一些，整理成需求
